### PR TITLE
[DOC-193] [2.20.2] YBA and YBDB compatibility section

### PR DESCRIPTION
**Pull request by aishwarya24**
@netlify /preview/releases/yba-releases/

### CODE REVIEW

File: docs/content/stable/yugabyte-platform/install-yugabyte-platform/prerequisites/default.md [hunk 1]
```diff
@@ -76,6 +76,16 @@ A node running YugabyteDB Anywhere is expected to meet the following requirement
...
+## Supported YugabyteDB versions
+
+YugabyteDB Anywhere aims to support _all_ officially supported versions of YugabyteDB.
+
+YBA {{< yb-version version="stable" format="short">}} supports deploying the following versions of YugabyteDB:
+
+- v2.14 series: 2.14.0 to 2.14.15
+- v2.18 series: 2.18.0 to 2.18.5
+- v2.20 series: 2.20.0 to 2.20.1.3
+
```
Comment (ddhodge) [inline docs/content/stable/yugabyte-platform/install-yugabyte-platform/prerequisites/default.md:88]: add an xref to https://docs.yugabyte.com/stable/yugabyte-platform/manage-deployments/upgrade-software/

"For information on managing YugabyteDB releases and upgrading universes, refer to [Upgrade the YugabyteDB software](../../manage-deployments/upgrade-software/)."

File: docs/content/stable/yugabyte-platform/install-yugabyte-platform/prerequisites/default.md [hunk 2]
```diff
@@ -76,6 +76,16 @@ A node running YugabyteDB Anywhere is expected to meet the following requirement
 - 8 GB memory
 - 200 GB disk space
 
+## Supported YugabyteDB versions
+
+YugabyteDB Anywhere aims to support _all_ officially supported versions of YugabyteDB.
```
Comment (ddhodge) [inline docs/content/stable/yugabyte-platform/install-yugabyte-platform/prerequisites/default.md:81]:
```suggestion
YugabyteDB Anywhere supports _all_ officially supported versions of YugabyteDB.
```

File: docs/content/stable/yugabyte-platform/install-yugabyte-platform/prerequisites/default.md [hunk 3]
```diff
@@ -76,6 +76,16 @@ A node running YugabyteDB Anywhere is expected to meet the following requirement
...
+
+YBA {{< yb-version version="stable" format="short">}} supports deploying the following versions of YugabyteDB:
```
Comment (ddhodge) [inline docs/content/stable/yugabyte-platform/install-yugabyte-platform/prerequisites/default.md:83]:
```suggestion
You can use YBA {{< yb-version version="stable" format="short">}} to deploy the following versions of YugabyteDB:
```

**Review by ddhodge — COMMENTED** [review]
Do we need to adjust this list for each backported version? Does YBA 2.18 support deploying 2.20 universes?

**Comment by aishwarya24** [comment]
> Do we need to adjust this list for each backported version? Does YBA 2.18 support deploying 2.20 universes?

We can't add it for the YBA 2.18 docs, nor YBA 2.16, because we did not perform compatibility testing for those versions.

File: docs/content/stable/yugabyte-platform/install-yugabyte-platform/prerequisites/installer.md [hunk 4]
```diff
@@ -75,6 +75,18 @@ A node running YugabyteDB Anywhere is expected to meet the following requirement
 - 215 GB disk space
 - x86 architecture
 
+## Supported YugabyteDB versions
+
+YugabyteDB Anywhere aims to support _all_ officially supported versions of YugabyteDB.
```
Comment (ddhodge) [inline docs/content/stable/yugabyte-platform/install-yugabyte-platform/prerequisites/installer.md:80]: aims? and kubernetes below

**Review by ddhodge — APPROVED** [review]
LGTM

**Pull request merged**